pkg/amqp-ext/Client/AmqpDriver.php

```diff
@@ -3,14 +3,14 @@
 namespace  Enqueue\AmqpExt\Client;
```

Comment:
@ASKozienko could you please move amqp client related to Enqueue\Client\Amqp namespace?